microsite/blog/2023-07-08-stash-adopter-post.mdx

```diff
@@ -0,0 +1,185 @@
+
+
+![alt_text](assets/2023-07-08/image1.png)
+
+
+![alt_text](assets/2023-07-08/image2.png)
+
+
+_CODEOWNER cards for service ownership and recent changes that appear on a service's overview page._
+
+
+## Integration with SaaS products
+
+Backstage offers a catalog of integrations with many popular SaaS products. With Elevate, we've integrated plugins for monitoring infrastructure and user experience, visualizing process backlog and bottlenecks, as well as giving insights into data that teams have not yet been able to see for services they support. These integrations allow for seamless data collection and aggregation, transforming Elevate into the centralized hub for application and team health data.
+
+The key to making your service catalog scalable is to ensure you aren't requiring the duplication of data away from its source; Elevate utilizes dashboards and charts to view the data stored in the remote sources. When aggregating DORA metrics we are able to leverage our SaaS tools API within Elevate's service layer to pull data and transform it for our custom dashboards.
+
+
+### Visible cohesive monitoring
+
+At Stash, we collect performance metrics, resource utilization, and proactive alerts with DataDog and Sentry; these tools can also monitor and track API errors and customer-facing issues that would normally go undetected. To deliver the best possible experience for Stash's customers, engineers need to be able to easily monitor errors on customer devices to solve issues before they are reported.
+
+Sentry's plugin easily surfaces uncaught errors within services. DataDog's plugin allows for plug-and-play visualizations of charts or graphs to help support services, teams and beyond. Elevate integrates these insights to monitor the overall health of the application.
+
+
+### Centralized incident management
+
+PagerDuty has been Stash's standard for on-call activities and paging while FireHydrant has more recently been adopted as a part of a more comprehensive incident management process. Together, we use these incident management platforms to establish incident response workflows and communication channels dedicated to each service.
+
+PagerDuty's plugin allows for quick reference to the individual who is actively on-call for a service while FireHydrant's plugin lets you track open incidents that may be impacting a service and even trigger new incidents directly from within the service overview page in Elevate. Integrating these plugins allows Elevate to serve as a centralized location for the incident management process, helping our engineers quickly gather the necessary context of an issue and drive faster resolution 
+
+
+![alt_text](assets/2023-07-08/image3.png)
+
+
+![alt_text](assets/2023-07-08/image4.png)
+
+
+_FireHydrant and PagerDuty cards that appear on a service's overview page._
```

Comment:
similar feedback to the codeowners section. i think image3 here is for code owners, and maybe since this is the fire hydrant section, it meant to link to image 1?  just might want to check the screen shots against these inclusions?